servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ProtocolConfig.java

```diff
@@ -60,6 +61,21 @@ default String alpnId() {
     @Nullable
     KeepAlivePolicy keepAlivePolicy();
 
+    /**
+     * Get a {@link Map} which provides a hint for the initial settings for any h2 connection. Note that some settings
+     * may be ignored if not supported (e.g. push promise).
+     * @return a {@link Map} which provides a hint for the initial settings for any h2 connection. Note that some
+     * settings may be ignored if not supported (e.g. push promise).
+     */
+    Map<Character, Integer> initialSettings();
+
+    /**
+     * Provide a hint on the number of bytes that the flow controller will attempt to give to a stream for each
+     * allocation (assuming the stream has this much eligible data).
+     * @return number of bytes.
+     */
+    int flowControlQuantum();
```

Comment:
now that folks can control the initial flow control we should also allow for them to control flow control quantum. incase the flow control window is very big or small they may want to adjust this.

Reply:
Do you plan to add connection level flow-control in this PR? Without it, increased `INITIAL_WINDOW_SIZE` for streams won't have much effect.

Reply:
Yes I can add this. increased flow control windows will still have effect without it, just that single streams can exhaust connection window more easily without it.

Reply:
Actually we ported increasing the connection window to Netty (doubles it by default). I will add an option to extend even further in ServiceTalk but set to 0 by default.

https://github.com/netty/netty/blob/4.1/codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FrameCodec.java#L229-L243